src/test/java/teammates/test/cases/BaseTestCaseWithMinimalGaeEnvironment.java

```diff
@@ -14,6 +20,22 @@
 
     private LocalServiceTestHelper helper = new LocalServiceTestHelper();
 
+    /**
+     * Sets the environment of SystemProperty based on the URL.
+     */
+    @BeforeSuite
+    public void setUpEnvironment() {
```

Comment:
Have you tried to change the link to your staging server address (start with `https`)? TDoes the tests pass?

Reply:
Yes. Both the tests failing before now pass on the staging server